src/database/utilities/createRecord.js

```diff
@@ -145,7 +145,7 @@ const createRequisition = (
   const { name: orderTypeName, maxMOS, thresholdMOS } = orderType;
   const regimenData =
     program && program.parsedProgramSettings ? program.parsedProgramSettings.regimenData : null;
-  const daysToSupply = monthsLeadTime ? (monthsLeadTime + maxMOS) * 30 : 30;
+  const daysToSupply = monthsLeadTime + (maxMOS || 1) * 30;
```

Comment:
On the subject of defensive programming, should probably add a similar check for `monthsLeadTime`. It has a default value, but we can't guarantee that another component might not pass it a `NaN` or similar value also. Could make it consistent with:

`const daysToSupply = ((monthlyLeadTime || 0) + (maxMOS || 1)) * 30;`

Reply:
62b4f46be71e714d80e6de78047613566f625c15